### ISSUE TITLE
Load config error, permission denied and EnvironmentError

### ISSUE THREAD

**Issue by created-Bi** [huggingface/transformers#19420]
### System Info

_libgcc_mutex             0.1                        main    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
_pytorch_select           0.2                       gpu_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
aiohttp                   3.8.3                    pypi_0    pypi
aiosignal                 1.2.0                    pypi_0    pypi
async-timeout             4.0.2                    pypi_0    pypi
asynctest                 0.13.0                   pypi_0    pypi
attrs                     22.1.0                   pypi_0    pypi
blas                      1.0                         mkl    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
blessed                   1.19.1                   pypi_0    pypi
boto3                     1.16.7             pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
botocore                  1.19.9             pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bottleneck                1.3.4            py37hce1f21e_0  
brotli                    1.0.9                he6710b0_2  
brotlipy                  0.7.0           py37h27cfd23_1003    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bzip2                     1.0.8                h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates           2022.07.19           h06a4308_0  
certifi                   2022.6.15        py37h06a4308_0  
cffi                      1.14.3           py37h261ae71_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
charset-normalizer        2.0.4              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
click                     8.0.1              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cryptography              3.1.1            py37h1ba5d50_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cudatoolkit               10.2.89              hfd86e86_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cudnn                     7.6.5                cuda10.2_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cycler                    0.11.0             pyhd3eb1b0_0  
datasets                  2.5.2                    pypi_0    pypi
dbus                      1.13.18              hb2f20db_0  
dill                      0.3.5.1                  pypi_0    pypi
docutils                  0.14                     py37_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
elastic-transport         8.1.2                    pypi_0    pypi
elasticsearch             7.14.1                   pypi_0    pypi
et_xmlfile                1.1.0            py37h06a4308_0  
expat                     2.4.4                h295c915_0  
ffmpeg                    4.2.2                h20bf706_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
filelock                  3.7.1                    pypi_0    pypi
flask                     2.1.3                    pypi_0    pypi
fontconfig                2.13.1               h6c09931_0  
fonttools                 4.34.4                   pypi_0    pypi
freetype                  2.11.0               h70c0345_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
frozenlist                1.3.1                    pypi_0    pypi
fsspec                    2022.8.2                 pypi_0    pypi
gevent                    21.12.0                  pypi_0    pypi
gevent-websocket          0.10.1                   pypi_0    pypi
giflib                    5.2.1                h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
glib                      2.69.1               h4ff587b_1  
gmp                       6.2.1                h2531618_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
gnutls                    3.6.15               he1e5248_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
gpustat                   1.0.0                    pypi_0    pypi
greenlet                  1.1.2                    pypi_0    pypi
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
huggingface-hub           0.8.1                    pypi_0    pypi
icu                       58.2                 he6710b0_3  
idna                      2.10                       py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
importlib-metadata        4.11.3           py37h06a4308_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
intel-openmp              2020.2                      254    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
itsdangerous              2.1.2                    pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
jmespath                  0.10.0                     py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
joblib                    1.0.1              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jpeg                      9e                   h7f8727e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
kiwisolver                1.4.4                    pypi_0    pypi
lame                      3.100                h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lcms2                     2.12                 h3be6417_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ld_impl_linux-64          2.33.1               h53a641e_7    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libedit                   3.1.20191231         h14c3975_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libffi                    3.3                  he6710b0_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libgcc-ng                 9.1.0                hdf63c60_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libgfortran-ng            7.5.0               ha8ba4b0_17  
libgfortran4              7.5.0               ha8ba4b0_17  
libidn2                   2.3.2                h7f8727e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libopus                   1.3.1                h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libpng                    1.6.37               hbc83047_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libstdcxx-ng              9.1.0                hdf63c60_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libtasn1                  4.16.0               h27cfd23_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libtiff                   4.2.0                h85742a9_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libunistring              0.9.10               h27cfd23_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libuuid                   1.0.3                h7f8727e_2  
libuv                     1.40.0               h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libvpx                    1.7.0                h439df22_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libwebp                   1.2.2                h55f646e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libwebp-base              1.2.2                h7f8727e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libxcb                    1.15                 h7f8727e_0  
libxml2                   2.9.10               hb55368b_3  
lpips                     0.1.4                    pypi_0    pypi
lz4-c                     1.9.3                h295c915_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.5.2                    pypi_0    pypi
matplotlib-base           3.5.1            py37ha18d171_1  
mkl                       2020.2                      256    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl-service               2.3.0            py37he8ac12f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_fft                   1.2.0            py37h23d657b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_random                1.1.1            py37h0573a6f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
multidict                 6.0.2                    pypi_0    pypi
multiprocess              0.70.13                  pypi_0    pypi
munkres                   1.1.4                      py_0  
ncurses                   6.2                  he6710b0_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nettle                    3.7.3                hbbd107a_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ninja                     1.10.1           py37hfd86e86_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nltk                      3.6.2              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numexpr                   2.7.3            py37hb2eb853_0  
numpy                     1.19.2           py37h54aff64_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numpy-base                1.19.2           py37hfa32c7d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nvidia-ml-py              11.495.46                pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openh264                  2.1.1                h4ff587b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
openpyxl                  3.0.9              pyhd3eb1b0_0  
openssl                   1.1.1q               h7f8727e_0  
opentsne                  0.6.2                    pypi_0    pypi
packaging                 21.3               pyhd3eb1b0_0  
pandas                    1.3.5                    pypi_0    pypi
pcre                      8.45                 h295c915_0  
pillow                    9.2.0                    pypi_0    pypi
pip                       20.2.4           py37h06a4308_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
psutil                    5.9.2                    pypi_0    pypi
pyarrow                   9.0.0                    pypi_0    pypi
pycparser                 2.20                       py_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyopenssl                 19.1.0                     py_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyparsing                 3.0.9            py37h06a4308_0  
pyqt                      5.9.2            py37h05f1152_2  
pysocks                   1.7.1                    py37_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python                    3.7.10               hdb3f193_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python-dateutil           2.8.1                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytorch-mutex             1.0                         cpu    pytorch-nightly
pytz                      2022.1           py37h06a4308_0  
pyyaml                    6.0                      pypi_0    pypi
qt                        5.9.7                h5867ecd_1  
readline                  8.0                  h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
regex                     2021.7.6         py37h7f8727e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
requests                  2.28.1                   pypi_0    pypi
responses                 0.18.0                   pypi_0    pypi
s3transfer                0.3.3              pyhd3eb1b0_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sacremoses                0.0.53                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
seaborn                   0.11.2             pyhd3eb1b0_0  
sentence-transformers     2.0.0                    pypi_0    pypi
sentencepiece             0.1.96                   pypi_0    pypi
setuptools                50.3.0           py37h06a4308_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sip                       4.19.8           py37hf484d3e_0  
six                       1.15.0                     py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sqlite                    3.33.0               h62c20be_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
threadpoolctl             3.1.0                    pypi_0    pypi
tk                        8.6.10               hbc83047_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tokenizers                0.9.4                    pypi_0    pypi
torch                     1.11.0+cu113             pypi_0    pypi
torchaudio                0.11.0+cu113             pypi_0    pypi
torchvision               0.12.0+cu113             pypi_0    pypi
tornado                   6.1              py37h27cfd23_0  
tqdm                      4.64.0           py37h06a4308_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
transformers              4.2.1                    pypi_0    pypi
typing_extensions         3.10.0.0           pyh06a4308_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
urllib3                   1.26.10                  pypi_0    pypi
wcwidth                   0.2.5                    pypi_0    pypi
werkzeug                  2.0.1                    pypi_0    pypi
wget                      3.2                      pypi_0    pypi
wheel                     0.35.1                     py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
x264                      1!157.20191217       h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xlrd                      2.0.1                    pypi_0    pypi
xlsxwriter                3.0.3              pyhd3eb1b0_0  
xlwt                      1.3.0                    pypi_0    pypi
xxhash                    3.0.0                    pypi_0    pypi
xz                        5.2.5                h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
yarl                      1.8.1                    pypi_0    pypi
zhconv                    1.4.3                    pypi_0    pypi
zipp                      3.5.0              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zlib                      1.2.11               h7b6447c_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zope-event                4.5.0                    pypi_0    pypi
zope-interface            5.4.0                    pypi_0    pypi
zstd                      1.4.9                haebb681_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

![image](https://user-images.githubusercontent.com/28759055/194686581-e33e8adb-3f13-41b2-a146-ee796e3c8632.png)

![image](https://user-images.githubusercontent.com/28759055/194686588-3c3c2e81-97f6-4b29-8dcd-8a34d6be97db.png)

### Expected behavior

load the config correctly

**Comment by LysandreJik** [comment]
Hello! It seems like you don't have read access into the cache generated by huggingface?

You're getting a permission denied to read the file.
